frontend/packages/ceph-storage-plugin/src/utils/kebab-actions.ts

```diff
@@ -0,0 +1,19 @@
+import { DeleteSnapshot } from './delete-snapshot-workflow';
+import { K8sKind } from '@console/internal/module/k8s';
+import { KebabAction } from '@console/internal/components/utils';
+import { PersistentVolumeClaimModel } from '@console/internal/models/index';
+import { RestorePVC } from './restore-pvc-workflow';
+import { SnapshotPVC } from './snapshot-workflow';
+import { VolumeSnapshotModel } from '../models';
+
+export const getKebabActionsForKind = (resourceKind: K8sKind): KebabAction[] => {
+  if (resourceKind) {
```

Comment:
```suggestion
  if (!resourceKind) {
      return [];
  }
```
Early returns results in less computations